MapboxNavigation/NavigationMapView.swift

```diff
@@ -46,6 +46,14 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
      */
     public var tapGestureDistanceThreshold: CGFloat = 50
     
+    /**
+     A collection of road classes for which a congestion level substitution should occur.
+     
+     For any road class included in the `trafficOverrideRoadClasses`, all route segments with an `.unknown` traffic congestion level and a matching `MapboxDirections.RoadClass`
+     will be replaced with the `.low` congestion level.
```

Comment:
This seems to state that the actual route model would change, but I think what would happen is that the segment would be styled similarly to segments with the `.low` congestion level, right?

Reply:
Yes, for that specific `RoadClasses` congestion level will be replaced from `.unknown` to `.low`. If you feel like this comment in not clear enough I'm open to suggested improvements.